bigquery/google/cloud/bigquery/schema.py

```diff
@@ -41,9 +41,13 @@ def __init__(self, name, field_type, mode='NULLABLE', description=None,
         self.field_type = field_type
         self.mode = mode
         self.description = description
-        self.fields = fields
+        self.fields = None if(fields is None) else tuple(fields)
```

Comment:
You can just use `tuple(fields)` here if you use `fields=()` as the default rather than `fields=None`

----

General Python advice, drop all superfluous parens (here I mean the parens around `(fields is None)`. (See [Google Python Style Guide][1].)

[1]: https://google.github.io/styleguide/pyguide.html#Parentheses